delphi-frontend/src/test/java/au/com/integradev/delphi/compiler/PredefinedConditionalsTest.java

```diff
@@ -368,4 +369,10 @@ void testLinuxOnlySupportsNextGenFeaturesBeforeDelphiRio() {
     assertThat(PredefinedConditionals.getConditionalDefines(Toolchain.DCCLINUX64, VERSION_RIO))
         .doesNotContainAnyElementsOf(NEXT_GEN_FEATURES);
   }
+
+  @Test
+  void testVersionsAfterDelphiAlexandriaShouldDefineLLVM() {
```

Comment:
For completeness, probably good to add a test to assert that versions before Alexandria and non-LLVM toolchains after Alexandria don't define it.